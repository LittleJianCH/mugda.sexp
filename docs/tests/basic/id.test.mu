(import "./id.mu" id)

id
(id (Pi ([A Type]) (-> A A)) id)

((id (Pi ([A Type]) (-> A A)) id)
 (Pi ([A Type]) (-> A A))
 (id (Pi ([A Type]) (-> A A)) id))
