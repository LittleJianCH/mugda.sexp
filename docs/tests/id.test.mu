(import "./id.mu" id)

((id id) (id id))
