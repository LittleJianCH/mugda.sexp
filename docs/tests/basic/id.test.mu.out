(lambda (A) (lambda (a) a))
(lambda (A) (lambda (a) a))
(lambda (A) (lambda (a) a))
